examples/material-gallery/build.rs

```diff
@@ -0,0 +1,6 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: MIT
+
+fn main() {
+    slint_build::compile("ui/index.slint").unwrap();
```

Comment:
We have `main.slint`, `app-window.slint`, `index.slint` - it would really be nice to reduce that set eventually :).